source/_integrations/adaptive_lighting.markdown

```diff
@@ -0,0 +1,257 @@
+It has several attributes that show the current light settings.
+The second switch is `switch.adaptive_lighting_sleep_mode_default` (with name `"default"`) that when activated, turns on "sleep mode".
+
+## Taking back control
+
+Although having your lights automatically adapt is great most of the time, there might be times at which you want to set the lights to a different color/brightness and keep it that way.
+For this purpose, the integration (when `take_over_control` is enabled) automatically detects whether someone (e.g., person toggling the light switch) or something (automation) changes the lights.
+If this happens *and* the light is already on, the light that was changed gets marked as "manually controlled" and the Adaptive Lighting component will stop adapting that light until it turns off and on again (or if you use the service call `adaptive_lighting.set_manual_control`).
+This mechanism works by listening to all `light.turn_on` calls and by noting that the component did not make the call.
+Additionally, there is an option to detect all state changes (when `detect_non_ha_changes` is enabled), so also changes to the lights that were not made by a `light.turn_on` call (e.g., through an app or via something outside of Home Assistant.)
+It does this by comparing a light's state to Adaptive Lighting's previously used settings.
+Whenever a light gets marked as "manually controlled", an `adaptive_lighting.manual_control` event is fired, such that one can use this information in automations.
+
+## Configuration
+
+This integration is both fully configurable through YAML _and_ the frontend. (**Configuration** -> **Integrations** -> **Adaptive Lighting**, **Adaptive Lighting** -> **Options**)
+Here, the options in the frontend and in YAML have the same names.
+
+```yaml
+# Example configuration.yaml entry
+adaptive_lighting:
+  lights:
+    - light.living_room_lights
+```
+
+{% configuration %}
+name:
+  description: The name to use when displaying this switch.
+  required: false
+  default: default
+  type: string
+lights:
+  description: List of light entities for Adaptive Lighting to control (may be empty).
+  required: false
+  type: list
+  default: []
+adapt_brightness:
+  description: Whether Adaptive Lighting should adjust brightness of lights (if supported by the light).
+  required: false
+  default: true
+  type: boolean
+adapt_color_temp:
+  description: Whether Adaptive Lighting should adjust color temperature of lights (if supported by the light).
+  required: false
+  default: true
+  type: boolean
+adapt_rgb_color:
+  description: Whether Adaptive Lighting should adjust RGB color of lights (if supported by the light).
+  required: false
+  default: true
+  type: boolean
+prefer_rgb_color:
+  description: Whether to use RGB color adjustment instead of native light color temperature.
+  required: false
+  default: false
+  type: boolean
```

Comment:
```suggestion
lights:
  description: List of light entities for Adaptive Lighting to control (may be empty).
  required: false
  type: list
  default: []
prefer_rgb_color:
  description: Whether to use RGB color adjustment instead of native light color temperature.
  required: false
  default: false
  type: boolean
```